src/coreComponents/physicsSolvers/fluidFlow/unitTests/testStatOutputController.cpp

```diff
@@ -0,0 +1,255 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2016-2024 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2024 Total, S.A
+ * Copyright (c) 2018-2024 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2023-2024 Chevron
+ * Copyright (c) 2019-     GEOS/GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+// Source includes
+#include "unitTests/fluidFlowTests/testCompFlowUtils.hpp"
+#include "unitTests/testingUtilities/TestingTasks.hpp"
+#include "mainInterface/initialization.hpp"
+#include "mainInterface/GeosxState.hpp"
+#include "mainInterface/ProblemManager.hpp"
+#include "common/format/table/TableData.hpp"
+#include "common/format/table/TableFormatter.hpp"
+#include "common/format/table/TableLayout.hpp"
+#include "physicsSolvers/fluidFlow/SinglePhaseStatistics.hpp"
+#include <hdf5.h>
+#include "dataRepository/Group.hpp"
+#include <filesystem>
+#include "fileIO/Outputs/TimeHistoryOutput.hpp"
+
+// TPL includes
+#include <gtest/gtest.h>
+#include <gtest/gtest-spi.h>
+
+using namespace geos;
+using namespace geos::dataRepository;
+using namespace geos::testing;
+
+CommandLineOptions g_commandLineOptions;
+
+namespace ComponentsGeneration
+{
```

Comment:
As you only have one test, I'm not sure we need this namespace.
Maybe it should go in `geos::testing`, but we may need a clear namespace policy.
`ComponentsGeneration` doesn't seem to fit.

Reply:
i removed the namespace and use `public ::testing::Test`